lib/melodica_inventory_web/controllers/admin/loan_controller.ex

```diff
@@ -20,7 +20,9 @@ defmodule MelodicaInventoryWeb.Admin.LoanController do
   defp apply_filters(loans, %{"name" => name, "user_id" => user_id}) do
     loans
     |> Enum.filter(fn loan ->
-      (name == "" || String.contains?(loan.item.name, name)) && (user_id == "" || loan.user_id == String.to_integer(user_id))
+      (name == "" ||
```

Comment:
Is this changed, because of the max line length rule? Given that the line fits on the github diff UI I would vote for increasing the max length to something like 120